mount.js

```diff
@@ -53,7 +53,45 @@ export function mountJsx (selector, ComponentToMount, additionalProps = {})
         }
         catch (e)
         {
-            // ignore
+            console.error("Could not mount component", e);
+        }
+    }
+}
+
+
+/**
+ * Mounts the component as JSX Component.
+ * The matched node must contain the data as JSON encoded content.
+ * The content will be automatically encapsulated in a unistore store.
+ *
+ * @param {string} selector
+ * @param {preact.Component} ComponentToMount
+ * @param {Object} additionalProps
+ */
+export function mountJsxWithStore (selector, ComponentToMount, additionalProps = {})
```

Comment:
Separate function instead of boolean flag to allow tree-shaking unistore away, if it is not used.